cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -182,13 +254,63 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_index_with_ends_with() {
+    executePreparationQueries {
+      val a = (0 to 100).map { i => "CREATE (:Person)" }.toList
+      val b = (0 to 300).map { i => s"CREATE (:Person {firstname: '$i'})" }.toList
+      a ++ b
+    }
+
+    sampleAllIndexesAndWait()
+
+    profileQuery(
+      title = "Suffix search using `ENDS WITH` (single-property index)",
+      text =
+        "The `ENDS WITH` predicate on `person.firstname` in the following query will use the `Person(firstname)` index, if it exists. " +
+          "All values stored in the `Person(firstname)` index will be searched, and entries ending with `'hn'` will be returned. " +
+          "This means that although the search will not be optimized to the extent of queries using `=`, `IN`, `>`, `<` or `STARTS WITH`, it is still faster than not using an index in the first place. " +
```

Comment:
Where can I learn about how queries using "`=`, `IN`, `>`, `<`, or `STARTS WITH` are optimized but queries using `ENDS WITH` are not?